src/main/java/ch/njol/skript/expressions/ExprFireworkEffect.java

```diff
@@ -87,10 +87,10 @@ protected FireworkEffect[] get(Event e) {
 		FireworkEffect.Builder builder = FireworkEffect.builder().with(type);
 		
 		for (Color colour : color.getArray(e))
-			builder.withColor(colour.asBukkitColor());
+			builder.withColor(colour.asDyeColor().getFireworkColor());
```

Comment:
Transforming the color into a dye color removes the RGB support Fireworks and SkriptColor can support.

This should be described in the description of the syntax that the color comparison will only work on vanilla Minecraft dye colors, or support should be added for RGB color comparison.

Reply:
Hey, so I'm just curious what you think would be best, 
adding a note in the description explaining the `%color%` will only work when comparing to a vanilla mc dye colors
or somehow adding support for RGB in the syntax?